golem/client.py

```diff
@@ -385,6 +383,18 @@ def start_network(self):
             task_finished_cb=self._task_finished_cb,
         )
 
+        def get_performance_values():
+            values = self.environments_manager.get_performance_values()
+            new_env_manager = self.task_server.task_keeper.new_env_manager
+            for new_env in new_env_manager.environments():
+                env_id = new_env.metadata().id
+                value = new_env_manager.get_cached_performance(env_id)
+                if value:

```

Comment:
```suggestion
                if value is not None:
```
Performance might be `0.0` and we want to include it then.